osquery/tables/system/windows/tpm_info.cpp

```diff
@@ -19,8 +19,7 @@ QueryData genTpmInfo(QueryContext& context) {
   std::string wmiclass{"SELECT * FROM Win32_Tpm"};
 
   BSTR wminamespace = ::SysAllocString(L"root\\cimv2\\Security\\MicrosoftTpm");
-  if (wminamespace == NULL) {
-    LOG(WARNING) << "failed to allocate string";
+  if (wminamespace == nullptr) {

```

Comment:
@theopolis this change seems to have broken the build_x86 / build_macos check. I can't see the logs to check what exactly went wrong. Any thoughts here?

Reply:
That build is just flaky (see https://github.com/osquery/osquery/issues/7192), it shouldn't be related to this change.